library/alloc/src/string.rs

```diff
@@ -76,6 +76,9 @@ use crate::vec::Vec;
 /// contents of the string. It has a close relationship with its borrowed
 /// counterpart, the primitive [`str`].
 ///
+/// `String` is a heap-allocated buffer. As such, it can be resized and modified
+/// freely. See the [Representation](#representation) section below for more.
```

Comment:
```suggestion
/// `String` uses a heap-allocated buffer. As such, it can be resized and modified
/// freely. See the [Representation](#representation) section below for more.
```

Reply:
Revised the whole paragraph.